presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix hanging join operator when spill is enabled and the probe side finishes before the hash builder starts.
```

Comment:
`Fix the join operator to prevent hanging when spill is enabled and the probe side finishes before the hash builder starts`